### PR TITLE
Add script to push an init action module to GCS.

### PR DESCRIPTION
**Pull request by functicons**

### CODE REVIEW

File: push-to-gcs.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script is intended to be used by the Dataproc team to push init action
+# modules (e.g., kafka) to the Dataproc init actions GCS bucket. It performs
+# necessary validations to make sure the repo is in good shape before pushing.
+#
+# Usage: ./push-to-gcs.sh <head> <module>
```
Comment (medb) [inline push-to-gcs.sh]: `head` -> `git-commit` or `gir-ref`

File: push-to-gcs.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#
+# Example: ./push-to-gcs.sh 119952 kafka
+
+set -euo pipefail
+
+[[ $# -eq 2 ]] || {
+    echo "Usage: ./push-to-gcs.sh <head> <module>"
```
Comment (medb) [inline push-to-gcs.sh]: ditto

File: push-to-gcs.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```
Comment (medb) [inline push-to-gcs.sh]: Should we run integration test before pushing changes to GCS as well?

Comment (functicons) [reply push-to-gcs.sh]: Currently integration test are running against the published init actions. They cannot test local changes. We need to think about how to change this later.

**Pull request merged**